db/migrate/20160414094300_change_capacity_to_hash_from_persistent_volume.rb

```diff
@@ -0,0 +1,56 @@
+class ChangeCapacityToHashFromPersistentVolume < ActiveRecord::Migration[5.0]
+  class ContainerVolume < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class PersistentVolume < ContainerVolume
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :capacity
+
+    IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+    def self.parse_iec_number(value)
```

Comment:
@Fryguy the method shouldn't be part of a model, it's not something we modeled or a format we came up with. This is the implementation of the IEC format and belongs more to a utility code. If there's anything wrong with this code I don't want to remember to fix it twice.

In fact the data that we store in the DB is not something that we came up with and "frozen" in time, but rather it's a IEC value that once we stored blindly as string and now we want instead to parse.